experimental/pgbouncer_with_vault/install.sh

```diff
@@ -55,12 +55,15 @@ sleep 60
 #=================================================================================================
 # SETUP VAULT
 #-------------------------------------------------------------------------------------------------
-git clone --single-branch --branch v0.1.2 https://github.com/hashicorp/vault-helm.git
+git clone --single-branch --branch master https://github.com/hashicorp/vault-helm.git
+if [ "$SINGLE_NODE_CLUSTER" = true ]; then
```

Comment:
This would fail for some tests. In bash it is recommended to compare either the numeric value or the string value, but otherwise avoid booleans. (in this case you are actually comparing strings).

See this tool to lint and order your bash scripts. It has plugins for vscode and other environments... 
https://github.com/koalaman/shellcheck

Reply:
ok thanks will check this.

Reply:
This is done.